### ISSUE TITLE
Doesn't Recognize Github Format

### ISSUE THREAD

**Issue by shellscape** [steelbrain/package-deps#345]
`apm` allows us to install from github (thought it doesn't support tags for some reason). Just dropping a note in case anyone else runs into this that `package-deps` doesn't support the github dep format.

**Comment by steelbrain** [comment]
Hello! This sounds like a thing I'd be happy to accept a PR for

**Comment by shellscape** [comment]
yeah it's a catch-22: we could implement this but none of the packages using this one would likely updated given the looming Dec 22nd end of life date for Atom. I wanted to put it in issues here so if people searched they'd have somewhat an answer.
